app/javascript/stylesheets/dashboard.scss

```diff
@@ -55,7 +55,7 @@ ul.collections {
   list-style-type: none;
 }
 
-#workTypeModal {
+.work-types {
```

Comment:
Prefer classes to IDs. This lets us use this same style in both the work type modal and the work form.